man/systemd.exec.xml

```diff
@@ -3690,6 +3690,15 @@ StandardInputData=V2XigLJyZSBubyBzdHJhbmdlcnMgdG8gbG92ZQpZb3Uga25vdyB0aGUgcnVsZX
           system.</para></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>$TRIGGER_REASON</varname></term>
+
+          <listitem><para>If the unit was activated dynamically (e.g.: a corresponding path unit), the reason
+          for the activation will be listed in this variable. Note that this is provided in a best-effort way.
+          For example, there might be multiple triggers firing at any given time, and only one will be reported,
+          and there is no guarantee on which one it will be.</para></listitem>
+        </varlistentry>
```

Comment:
maybe emphasize that this is *unreliable*. i.e. say

"Because of this, this variable is primarily informational, i.e. useful for debugging purposes, is lossy, and should not be relied upon to propagate a comprehensive reason for activation."

Reply:
I'll also qualify it, because if there's only one trigger configured, then it will be reliable, so that only applies when there are multiple ones

Reply:
nah, a service unit can always be started manually instead of relying on triggers. Thus you *always* must expect two ways of activation even if you only have one trigger configured: manual + whatever you added.

Reply:
Right, but in that case there will be nothing set. So the env var will still be reliable.